app/javascript/components/CurrentCrypto.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const CurrentCrypto = props => {
+
+
+  return(
+    <div id='CurrentCrypto'>
+      <img
+        className='homepageCryptoPic'
+        src='https://upload.wikimedia.org/wikipedia/commons/thumb/4/43/John_Oliver_November_2016.jpg/1200px-John_Oliver_November_2016.jpg'
```

Comment:
license wise, I think you want to attribute wikipedia for this image - https://en.wikipedia.org/wiki/Wikipedia:Image_use_policy